layouts/authors/terms.html

```diff
@@ -0,0 +1,6 @@
+<h1>Authors</h1>
```

Comment:
This provides style for an `/authors` page. It's basic now, so I can create a follow on story to design and style it.